pkg/agent/storage/bbolt/stores_bbolt.go

```diff
@@ -0,0 +1,40 @@
+package agentbbolt
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+
+	"github.com/kolide/launcher/pkg/agent/storage"
+	"github.com/kolide/launcher/pkg/agent/types"
+	"go.etcd.io/bbolt"
+)
+
+// MakeStores creates all the KVStores used by launcher
+func MakeStores(logger log.Logger, db *bbolt.DB) (map[storage.Store]types.KVStore, error) {
```

Comment:
.... just a thought, I wonder if we can live without this and just let knapsack lazily create the stores if they don't exist